docs/http-api/index.rst

```diff
@@ -17,8 +17,8 @@ The following webserver related configuration items are available:
 * :ref:`setting-webserver`: If set to anything but 'no', a webserver is launched.
 * :ref:`setting-webserver-address`: Address to bind the webserver to. Defaults to 127.0.0.1, which implies that only the local computer is able to connect to the nameserver! To allow remote hosts to connect, change to 0.0.0.0 or the physical IP address of your nameserver.
```

Comment:
this should probably mention that the address can be a UNIX socket path

Reply:
Adjusted.